service/src/main/java/org/kiwiproject/champagne/model/Component.java

```diff
@@ -0,0 +1,29 @@
+package org.kiwiproject.champagne.model;
+
+import java.time.Instant;
+
+import lombok.Builder;
+import lombok.Getter;
+
+/**
+ * Representation of a deployable component in the system.
+ */
+@Builder
+@Getter
+public class Component {
+    
+    Long id;
```

Comment:
Note that if you do _not_ add `@Value`, then these fields should be made `private final` (`@Value` does that, but a plain `@Getter` does not)